python/py-regions/Portfile

```diff
@@ -7,7 +7,7 @@ set _name           regions
 set _n              [string index ${_name} 0]
 
 name                py-${_name}
-version             0.2
+version             0.3
 categories-append   science
 platforms           darwin
 maintainers         {gmail.com:Deil.Christoph @cdeil} openmaintainer
```

Comment:
If you can no longer maintain this, you should remove yourself as the maintainer as well.

Reply:
I'll try to maintain this one and at least update it for new versions like I did here, it's an important one / dependency for other packages.